python/lsst/obs/decam/ingestCalibs.py

```diff
@@ -2,39 +2,16 @@
 
 import collections
 import re
-
+from glob import glob
 from lsst.pipe.tasks.ingestCalibs import CalibsParseTask
+from lsst.pipe.tasks.ingestCalibs import IngestCalibsTask
+from lsst.pipe.tasks.ingestCalibs import IngestCalibsConfig, IngestCalibsArgumentParser
+from lsst.pex.config import Field
 
 
 class DecamCalibsParseTask(CalibsParseTask):
-
-    def getInfo(self, filename):
```

Comment:
I don't think this override can be removed if we want to continue ingesting `defect` the same way before this ticket.  Would it hinder ingesting `bias`/`flat` in the new way?